SpeechResponder/eddi.json

```diff
@@ -1319,7 +1319,7 @@
+      "script": "{_ Mission check station }\r\n{_ Check what missions you have for the current station }\r\n\r\n\r\n{if missionsCount > 0:\r\n    {set current_station to state.eddi_context_body_name}\r\n    {set active_missions to 0}\r\n    {set claim_missions to 0}\r\n    {set complete_missions to 0}\r\n    {set depot_missions to 0}\r\n\r\n    {if current_station && current_station != \"\":\r\n        {for mission in missions:\r\n            {if mission.destinationstation = current_station ||\r\n                (mission.originstation = current_station && find(mission.type, \"Delivery\") > -1):\r\n                {if mission.status = \"Active\":\r\n                    {if find(mission.type, \"Collect\") > -1 || find(mission.type, \"Delivery\") > -1:\r\n                        {set depot_missions to depot_missions + 1}\r\n                    |else:\r\n                        {set active_missions to active_missions + 1}\r\n                    }\r\n                |elif mission.status = \"Claim\":\r\n                    {set claim_missions to claim_missions + 1}\r\n                |elif mission.status = \"Complete\":\r\n                    {set complete_missions to complete_missions + 1}\r\n                }\r\n            }\r\n        }\r\n    }\r\n\r\n    {set total to active_missions + claim_missions + complete_missions + depot_missions}\r\n    {if total > 0:\r\n        You have\r\n        {if depot_missions > 0:\r\n            {depot_missions} depot\r\n            {if active_missions > 0 && claim_missions > 0: ,\r\n            |elif active_missions > 0 && complete_missions > 0: ,\r\n            |elif claim_missions > 0 && complete_missions > 0: ,\r\n            |elif active_missions > 0 || claim_missions > 0 || complete_missions > 0: and }\r\n        }\r\n        {if active_missions > 0:\r\n            {active_missions} active\r\n            {if claim_missions > 0 && complete_missions > 0: ,\r\n            |elif claim_missions > 0 || complete_missions > 0: and }\r\n        }\r\n        {if claim_missions > 0:\r\n            {claim_missions} claimable\r\n            {if complete_missions > 0: and }\r\n        }\r\n        {if complete_missions > 0:\r\n            {complete_missions} completed\r\n        }\r\n        {if total = 1: mission |else: missions} {Occasionally(3,\"located\")} at this station.\r\n    }\r\n}\r\n",
```

Comment:
Perhaps consider using `{station.name}` in addition to `state.eddi_context_body_name` to determine the station that we are at? Testing while parked in a station orbiting a body and finding `state.eddi_context_body_name` not set to the name of the station.

Reply:
Fixed.